csharp/developer/OpenStudio/OpenStudio.csproj.in

```diff
@@ -1,70 +1,80 @@
-<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="${CSHARP_TOOLS_VERSION}" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">${CSHARP_PLATFORM}</Platform>
-    <ProductVersion>9.0.30729</ProductVersion>
-    <SchemaVersion>2.0</SchemaVersion>
-    <ProjectGuid>{63F7E86E-6A41-4033-A5C6-E41015F5AD4B}</ProjectGuid>
-    <OutputType>Library</OutputType>
-    <AppDesignerFolder>Properties</AppDesignerFolder>
-    <RootNamespace>OpenStudio</RootNamespace>
-    <AssemblyName>OpenStudio</AssemblyName>
-    <TargetFrameworkVersion>v3.5</TargetFrameworkVersion>
-    <FileAlignment>512</FileAlignment>
+    <TargetFrameworks Condition="'$(OS)' == 'Windows_NT'">NET45;netstandard2.0</TargetFrameworks>
```

Comment:
Out of curiosity, why did you pick netstandard2.0? 

(I had used `netcoreapp3.0` (but like I said, don't know C# much...))

https://docs.microsoft.com/en-us/dotnet/standard/frameworks

Reply:
Because .Net Standard works for both old .Net Framework and new .Net Core.
But for Windows, we add an additional target .Net Framework 4.5 because .Net Standard 2.0 only supports .Net Framework 4.6.1 or above. 

We could remove the 4.5 if everyone is using 4.6.1 or new .Net 5 or 6.

Reply:
@tijcolem If you did not notice this comment, I think the salient point is 

> .Net Standard 2.0 only supports .Net Framework 4.6.1 or above.